### ISSUE TITLE
Ask resident/contracted developer

### ISSUE THREAD

**Issue by schlessera** [WordPress/servehappy#33]

**Comment by felixarntz** [comment]
Just adding for now that this is probably only useful for the cases where a developer initially set up a site, but then never continuously maintained it (which happens lots of times for sure). A developer maintaining a site on a regular basis has probably (and hopefully) already made the switch to a current PHP version anyway.

**Comment by Mte90** [comment]
Agree, also on the profile on wp.org in the Extras tab there is a flag to add the user on the consultant list but I cannot find a link for it to check if has a filter by location.

**Comment by schlessera** [comment]
> A developer maintaining a site on a regular basis has probably (and hopefully) already made the switch to a current PHP version anyway.
...or consciously avoided PHP updates for valid reasons.

**Comment by felixarntz** [comment]
Per @nerrad's suggestion in today's meeting, let's go with this:
> While the steps below are something we believe any WordPress site owner can do, if you have an existing relationship with a developer/agency, you may want to also consult with them regarding the specific needs for your site.
